### PR TITLE
chore(e2e): Refactor authoring e2e tests

### PR DESCRIPTION
**Pull request by ioanpocol**

### CODE REVIEW

File: client/protractor-conf.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ exports.config = {
     framework: 'jasmine2',
     jasmineNodeOpts: {
         showColors: true,
-        defaultTimeoutInterval: 55000
+        defaultTimeoutInterval: 120000
```
Comment (amagdas) [inline client/protractor-conf.js]: Whoa, this is quite a bit timeout.

Comment (ioanpocol) [inline client/protractor-conf.js]: This is the time a test can run and because we have now larger tests suits, more time is needed.

**Comment by mugurrus** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

File: client/protractor-conf.js (hunk 1 above)
Comment (ioanpocol) [inline client/protractor-conf.js]: Anyway seams that larger tests are not a good idea, so we have to talk about it on retrospective meeting.

**Pull request merged**